### PR TITLE
Slowness Injection Local Support (Not ready)

### PR DESCRIPTION
**Pull request by Essoz**

### CODE REVIEW

**Comment by Essoz** [comment]
Will perform a test tommorrow morning.

**Comment by varshith15** [comment]
Can you please remove unwanted files @Essoz, so that we can add the node kill support onto this code, I mean it has changes to files other than slow.xsh please check.

**Comment by Essoz** [comment]
I have cleaned up files, please check.

File: resources/slowness/slow.xsh [hunk 1]
```diff
@@ -56,8 +57,11 @@ slow_vs_num = {1: cpu_slow,
                5: memory_contention,
                6: network_slow}
 
-def slow_inject(exp, host_id, secondaryip, secondarypids):
-    slow_vs_num[int(exp)](host_id, secondaryip, secondarypids)
+def slow_inject(exp, secondary_server_config, secondarypids):
+    print(secondaryip)
```
Comment (varshith15) [inline resources/slowness/slow.xsh]: @Essoz remove this please

**Comment by varshith15** [comment]
And also we can inject slowness on any of the nodes, not necessarily secondary nodes so maybe change the variable names? for clearer understanding @Essoz 
just ip rather than secondary_ip ?

File: resources/slowness/slow.xsh [hunk 2]
```diff
@@ -1,53 +1,54 @@
...
+def disk_slow(slow_server_config, slow_ip, slow_pids):
+    ssh -i ~/.ssh/id_rsa @(slow_ip) "sudo sh -c 'sudo mkdir /sys/fs/cgroup/blkio/db'"
+    ssh -i ~/.ssh/id_rsa @(slow_ip) "sudo sh -c 'sync; echo 3 > /proc/sys/vm/drop_caches'"
     lsblkcmd="8:32 524288"
-    ssh -i ~/.ssh/id_rsa @(host_id)@@(secondaryip) "sudo sh -c 'sudo echo $lsblkcmd > /sys/fs/cgroup/blkio/db/blkio.throttle.read_bps_device'"                 
-    ssh -i ~/.ssh/id_rsa @(host_id)@@(secondaryip) "sudo sh -c 'sudo echo $lsblkcmd > /sys/fs/cgroup/blkio/db/blkio.throttle.write_bps_device'"                                                                                                                         
-    for secondarypid in secondarypids.split():
-        ssh -i ~/.ssh/id_rsa @(host_id)@@(secondaryip) @("sudo sh -c 'sudo echo {} > /sys/fs/cgroup/blkio/db/cgroup.procs'".format(secondarypid))
+    ssh -i ~/.ssh/id_rsa @(slow_ip) "sudo sh -c 'sudo echo $lsblkcmd > /sys/fs/cgroup/blkio/db/blkio.throttle.read_bps_device'"                 
+    ssh -i ~/.ssh/id_rsa @(slow_ip) "sudo sh -c 'sudo echo $lsblkcmd > /sys/fs/cgroup/blkio/db/blkio.throttle.write_bps_device'"
```
Comment (varshith15) [inline resources/slowness/slow.xsh:31]: sudo echo $lsblkcmd won't work @Essoz it should be added as f"sudo echo {lsblkcmd}"

**Pull request merged**